tsconfig.base.json

```diff
@@ -10,7 +10,6 @@
     "moduleResolution": "Node",
     "noImplicitAny": true,
     "noImplicitReturns": true,
-    "preserveConstEnums": true,
```

Comment:
what was this doing?

Reply:
Another situation where the datastructure you use for I/O needs to have different types than the run-time data structure, for flexibility.  Once an external user persists an enum value, it's permanent.   We would call the persistent ones "PermId"s  (as in permanent) and have mappings in the I/O code to map to and from the runtime data structures.  The PermId enums would be in source files that were only associated with the file format and with big comments that said, never change values, remove items or reorder this!)